docs/compute-engine/build_arm.md

```diff
@@ -96,6 +96,23 @@ example program `lce_minimal` and benchmark program `benchmark_model`.
 In the `lib` folder, you can find the TensorFlow Lite static library
 `libtensorflow-lite.a` which includes the LCE customs ops.
 
+
+Note: on some systems the compiler is incorrectly named `aarch64-unknown-linux-gnu-gcc`
```

Comment:
Side note: If you want to make it stick out on the website, you can do something [like this](https://squidfunk.github.io/mkdocs-material/extensions/admonition/#removing-the-title).

Reply:
That looks nice. I updated it, where can i preview if it worked?

Reply:
The view deployment button should show it:
![Screenshot 2020-02-26 at 15 56 47](https://user-images.githubusercontent.com/13285808/75362434-9f666e00-58b0-11ea-8fec-1a7a98ffb707.png)

https://docs-ixzqp6vnn.now.sh/compute-engine/build_arm/